candy.js

```diff
@@ -1,4 +1,3 @@
-
 const spanSpam = () => {
   for (let i = 1; i < 7; i++) {
     let itm = document.getElementById("candySpan");
```

Comment:
we can make this dynamic for future reference and reusability -- vs. always targeting "candySpan" and always being 7

Reply:
const spam = (element, quantity) => { //blah }

Reply:
no change rn